bootstrap.d/dev-libs.yml

```diff
@@ -823,6 +823,40 @@ packages:
           DESTDIR: '@THIS_COLLECT_DIR@'
         quiet: true
 
+  - name: openssl
+    source:
+      subdir: ports
+      git: 'https://github.com/openssl/openssl.git'
+      tag: 'OpenSSL_1_1_1j'
+      version: '1.1.1j'
+    tools_required:
+      - system-gcc
+    pkgs_required:
+      - zlib
+    configure:
+      - args:
+        - '@THIS_SOURCE_DIR@/config'
+        - '--prefix=/usr'
+        - '--openssldir=/etc/ssl'
+        - '--libdir=lib'
+        - 'shared'
+        - 'zlib-dynamic'
+        environ:
+          CC: 'x86_64-managarm-gcc'
+          CXX: 'x86_64-managarm-g++'
+    build:
+      - args: ['make', '-j@PARALLELISM@']
+      # DIsable installing static libraries
+      - args: ['sed', '-i', '/INSTALL_LIBS/s/libcrypto.a libssl.a//', '@THIS_BUILD_DIR@/Makefile']
```

Comment:
Why do we do this?

Reply:
Do we want the static libraries? If yes, I’ll remove it. Otherwise, this disables installing it as stated above the sed.

Reply:
Installing the static libs doesn't hurt.

Reply:
But deps should link against the dynamic ones.

Reply:
Considering we pass disable-static to most packages, I’d say this keeps it consistent with the other packages

Reply:
typo